admob/src/ios/FADRequest.mm

```diff
@@ -24,16 +24,6 @@
 GADRequest *GADRequestFromCppAdRequest(AdRequest adRequest) {
   // Create the GADRequest.
   GADRequest *request = [GADRequest request];
-
-  // Test devices.
-  if (adRequest.test_device_id_count > 0) {
```

Comment:
Are these set elsewhere now?

Reply:
Yes, they were once per ad request, now they're part of RequestConfiguration structure. set globally through SetRequestConfigration().  No need to set them each and every time anymore!